scripts/generate_data_field_doc.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Lucca Baumgärtner <lucca.baumgaertner@fau.de>
+import inspect
+import os.path
+
+import pandas as pd
+
+from bdc import steps
```

Comment:
in macOS, bdc can not be found: 
![Screenshot 2024-01-10 at 11 08 15](https://github.com/amosproj/amos2023ws06-sales-lead-qualifier/assets/45459787/cc9112eb-394e-4eda-a3b6-b8001c3cd314)

Reply:
I think you're missing the `src` folder in your `PYTHONPATH` environment variable. That's why it can't be found when executing in the scripts folder.